### ISSUE TITLE
Add Proper Release Tags

### ISSUE THREAD

**Issue by aguckenber-chwy** [wiz-sec/charts#262]
Can proper release tags please be added for new releases of the helm charts? Its really difficult to hunt down the `values.yaml` documentation for an older release because their are no branch tags nor GitHub releases to reference.

**Comment by ariknem** [comment]
You mean that you want to see the versions under https://github.com/wiz-sec/charts/releases yes?

**Comment by aguckenber-chwy** [comment]
Yes, as well as Tags that point to the commit SHA from which the release came. It was quite difficult to find the helm values from your 1.0.0 release.

**Comment by ariknem** [comment]
Thanks. It's a reasonable ask. We'll take this internally with our devops :)

**Comment by aguckenber-chwy** [comment]
Any update on this? I saw another untagged version was released recently.

**Comment by danielkimuipath** [comment]
How can I publicly pull the HELM charts?

For example, What is the URL for pulling the wiz-kubernetes-integration helm chart?
